core/scripts/Voting.js

```diff
@@ -50,6 +56,17 @@ async function fetchCryptoComparePrice(request) {
 }
 
 async function fetchPrice(request) {
+  const config = SUPPORTED_IDENTIFIERS[request.identifier];
```

Comment:
You'll probably need to use `web3.utils.hexToUtf8(request.identifier)` to convert the identifier to a human readable string so you can look it up in `SUPPORTED_IDENTIFIERS`. I'm not 100% sure this will work, so I'd suggest using `console.log()` to see what it spits out.